### PR TITLE
DOC-2181: add fix documentation for TINY-10177 to the 6.8 release notes.

### PR DESCRIPTION
**Pull request by kemister85**
Ticket: DOC-2181

Changes:
* add fix documentation for TINY-10177 to the 6.8 release notes.

Pre-checks:
- [x] Branch prefixed with `feature/6/` or `hotfix/6/`
- [x] Changelog entry added

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

**Comment by vpyshnenko** [comment]
I'm not sure we should mention `isInSummary` predicate as it seems internal details and is not in publick API.

File: modules/ROOT/pages/6.8-release-notes.adoc [hunk 1]
```diff
@@ -233,6 +233,11 @@ waiting for confirmation on these new feature tickets.
 
 === Pressing Enter when focusing on the `details` component for an Accordion deletes its structure.
 // #TINY-10177
+Previously, clicking on the expand arrow of a `details` element would result in the editor selection to be external of the `summary` element when the host browser was Safari.
+
+As a consequence, the `isInSummary` predicate could not detect the selection at the beginning of the `details` element and would fail to toggle the Accordion component on **Enter** key press.
```
Comment (kemister85) [inline modules/ROOT/pages/6.8-release-notes.adoc:238]:
```suggestion
As a consequence, the plugin failed to detect if the selection was at the beginning of the `details` element, which resulted in a unsuccessful attempt to toggle the Accordion component when the **Enter** key is pressed
```

Comment (kemister85) [reply modules/ROOT/pages/6.8-release-notes.adoc:238]: @vpyshnenko see above suggestion for re-write

Comment (vpyshnenko) [reply modules/ROOT/pages/6.8-release-notes.adoc:238]: That's good one.

**Review by vpyshnenko — APPROVED** [review]

**Pull request merged**